pages/partials/jobSequenceResults.ejs

```diff
@@ -68,6 +68,9 @@
             <p class="mb-1">
                 Started at <%= job.start_date_formatted %><% if (job_sequence.user_uid) { %> by <%= job_sequence.user_uid %><% } %><% if (job_sequence.authn_user_uid != job_sequence.user_uid) { %> (really <%= job_sequence.authn_user_uid %>)<% } %><% if (job.finish_date_formatted) { %>, finished at <%= job.finish_date_formatted %><% } %>
             </p>
+            <p class="mb-1">
+                Please feel free to navigate away from this page. If any issues arise they will appear under the "Issues" page.
```

Comment:
I don't think it's generally true that all job issues will be filed on the Issues page. That's true for question issues arising from `test()`, but the job sequence page is also used for jobs like course syncing which will report errors on the Sync page. I'm tempted to just not add this message here at all? As a comparison, the GitHub Action page doesn't say anything about being able to navigate away, they just assume you know you can.

Reply:
I'd rather it only show up on sync, but since all the jobs use the same page maybe I'll just take it out for now.